src/databricks/labs/ucx/source_code/notebooks/sources.py

```diff
@@ -104,29 +104,75 @@ def name() -> str:
         return "notebook-linter"
 
 
-class FileLinter:
-    _EXT = {
+class ExtensionsHelper:
```

Comment:
Why do we need a whole class to wrap a single dictionary?

Reply:
Because we need the exact same dict elsewhere, and I expect this might grow, but I've simplified